<p>出于以下原因，不建议使用<code>File.deleteOnExit()</code> ：</p>
<ul>
  <li> 仅在JVM正常关闭的情况下才会发生删除，但在JVM崩溃或被杀死时则不会发生删除。 </li>
  <li> 对于每个文件处理程序，与处理程序关联的内存仅在进程结束时释放。 </li>
</ul>
<h2>Noncompliant Code Example</h2>
<pre>
File file = new File("file.txt");
file.deleteOnExit();  // 不合规的
</pre>

